syntax = "proto3";

import "data_transport_catalogue.proto";
import "map_renderer.proto";
import "transport_router.proto";

package t_c_srlz;

message TransportCatalogue {
    repeated d_t_c_srlz.Stop list_stop = 1;
    repeated d_t_c_srlz.Bus list_bus = 2;
    repeated d_t_c_srlz.StopsLenght list_stop_lenght = 3;
	r_s_srlz.RenderSettings render_settings = 4;
	t_r_srlz.TransportRouter t_r_ = 5;
} 